source/write/delete.txt

```diff
@@ -0,0 +1,212 @@
+
+.. literalinclude:: /includes/write/delete.scala
+    :language: scala
+    :dedent:
+    :start-after: start-db-coll
+    :end-before: end-db-coll
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Delete Operations
+-----------------
+
+You can perform delete operations in MongoDB by using the following methods:
+
+- ``deleteOne()``, which deletes *the first document* that matches the search criteria
+- ``deleteMany()``, which deletes *all documents* that match the search criteria
+
+Each delete method requires a **query filter** document, which specifies the
+search criteria that determine which documents to select for removal.
+To learn more about query filters, see the :ref:`scala-specify-query` guide.
+
+Delete One Document
+~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``deleteOne()`` method to remove a
+document in which the value of the ``name`` field is ``"Happy Garden"``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/write/delete.scala
+      :start-after: start-delete-many
+      :end-before: end-delete-many
+      :language: scala
+      :dedent:
+   
+   .. output::
+      :language: console
+      :visible: false
+
+      Deleted document count: 1
+      Completed
+
+Delete Multiple Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``deleteMany()`` method to remove all documents
+in which the value of the ``borough`` field is ``"Brooklyn"`` and the
+value of the ``name`` field is ``"Starbucks"``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/write/delete.scala
+      :start-after: start-delete-many
+      :end-before: end-delete-many
+      :language: scala
+      :dedent:
+   
+   .. output::
+      :language: console
+      :visible: false
+
+      Deleted document count: 3
+      Completed
+
+Customize the Delete Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``deleteOne()`` and ``deleteMany()`` methods optionally accept a
+``DeleteOptions`` parameter, which represents options you can use to
+configure the delete operation. If you don't specify any
+options, the driver performs the delete operation with default settings.
+
+The following table describes the setter methods that you can use to
+configure a ``DeleteOptions`` instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``collation()``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``hint()``
+     - | Specifies the index to use when matching documents.
+         For more information, see the :manual:`hint statement </reference/command/delete/#std-label-deletes-array-hint>`
```

Comment:
```suggestion
         For more information, see the :manual:`hint field </reference/command/delete/#std-label-deletes-array-hint>`description
```
S: Confused by the use of "statement", as the link leads to field descriptions. Suggested alternative wording, would apply to all of the following uses of "statement".

Reply:
Will make a change here